### ISSUE TITLE
default config filename `/etc/Rserv.conf` not consistient

### ISSUE THREAD

**Issue by jangorecki** [s-u/Rserve#48]
Huh, I've lost few hours on debugging that (in fact looking for an issue in docker layer)...
In the docs config file is defined as `/etc/Rserv.conf` while `run.Rserve` uses by default `"/etc/Rserve.conf"`.

**Comment by s-u** [comment]
Ah, sorry about that. The old `Rserv.conf` name is indeed confusing, but unfortunately it exists for historical reasons (`Rserv` was the original name before it became `Rserve`). Apparently, my finger memory was updated by the time I wrote the `run.Rserve` function - it was not intentional. It shows, however, a fundamental flaw - the `CONFIG_FILE` define is only available at C level, so the R code cannot use the same default (since `CONFIG_FILE` can override the default config location).

Ideally, I'd like to deprecate `Rserv.conf` and move to `Rserve.conf`. Also probably a good idea is to change the default in `run.Rserve()` to honor `CONFIG_FILE`.
